.gitignore

```diff
@@ -48,6 +48,10 @@ src/autoscaler/helpers/apis/scalinghistory/oas*gen.go
 # Autogenerated files
 go.work.sum
 
+# acceptance-test binaries
+src/acceptance/**/*.test
+src/acceptance/**/ginkgo_v2_*
```

Comment:
I was wondering binaries like `src/acceptance/app/app_linux_arm64.test` were not committed as well and found the reason here: https://github.com/cloudfoundry/app-autoscaler-release/blob/fc592710901c3f2c70aea6a3bafb3ddf4ed802cb/.gitignore#L30

I prefer to make it a bit more explicit so I listed `src/acceptance/**/*.test` again.

Reply:
But shouldnt every binary be not commited?

Reply:
Of course! :D Usually, binaries don't have file extensions like `*.test`. It was just a coincidence that ginkgo compiles the tests with a `.test`-extension. That's why they were ignored by the already existing blocklisting here  https://github.com/cloudfoundry/app-autoscaler-release/blob/fc592710901c3f2c70aea6a3bafb3ddf4ed802cb/.gitignore#L30

Since this behaviour was more like a "lucky coincidence", I made it a bit more explicit by adding `src/acceptance/**/*.test` to the blocklisting.